src/datasources/config-api/config-api.service.ts

```diff
@@ -23,26 +24,29 @@ export class ConfigApi implements IConfigApi {
 
   async getChains(limit?: number, offset?: number): Promise<Page<Chain>> {
     try {
-      const cacheKey = `chains`;
-      const field = `${limit}_${offset}`;
       const url = `${this.baseUri}/api/v1/chains`;
-      return await this.dataSource.get(cacheKey, field, url, {
-        params: {
-          limit,
-          offset,
+      return await this.dataSource.get(
+        CacheRouter.getChainsCacheDir(limit, offset),
+        url,
+        {
+          params: {
+            limit,
+            offset,
+          },
         },
-      });
+      );
     } catch (error) {
       throw this.httpErrorFactory.from(error);
     }
   }
 
   async getChain(chainId: string): Promise<Chain> {
     try {
-      const cacheKey = `${chainId}_chain`;
-      const field = '';
       const url = `${this.baseUri}/api/v1/chains/${chainId}`;
-      return await this.dataSource.get(cacheKey, field, url);
+      return await this.dataSource.get(
+        CacheRouter.getChainCacheDir(chainId),
```

Comment:
nitpick: for readability we could extract the `cacheDir` to a const.

```typescript
const url = `${this.baseUri}/api/v1/chains/${chainId}`;
const cacheDir = CacheRouter.getChainCacheDir(chainId);
return await this.dataSource.get(cacheDir, url);
```

Reply:
Changed in [76724b7](https://github.com/5afe/safe-client-gateway-nest/pull/285/commits/76724b74b6eeed8fb5ab522bf72045f6ffe0f730)